lqr_gain.json

```diff
@@ -0,0 +1 @@
+[[-1.3088867556805999,0.0001191892678373112,1.3088867556806705,-0.00011918926779724822],[0.00011918927234680743,-1.3103899397473247,-0.00011918927303761314,1.3103899397469996],[1.3803800171174212,1.3803800171172753,1.3803800171197116,1.3803800171194687],[-0.001515574521348988,15.223478696895773,0.001515574527058293,-15.223478696890872],[-15.204364665036639,0.0015155745088055663,15.204364665036682,-0.0015155745088410947],[1.467832505667062,-1.4678325056669157,1.467832505665548,-1.4678325056661783],[-1.9343752298533492,0.00018066645184270668,1.9343752298533707,-0.00018066645183605046],[0.0001806664539506274,-1.9366537481935868,-0.00018066645471739227,1.93665374819299],[1.5001813210207318,1.5001813210207886,1.5001813210209314,1.500181321020846],[-0.0001664349970135038,1.5437920568827428,0.00016643499707393234,-1.543792056882691],[-1.541693021951366,0.00016643499688754438,1.541693021951367,-0.00016643499688798023],[1.5773617029255342,-1.577361702925552,1.5773617029254496,-1.5773617029254652]]
```

Comment:
I think it's fine to keep this data file in the repo so we don't have to regenerate it very often.